src/apis/print.js

```diff
@@ -9,13 +13,32 @@ WebViewer(...)
     // you must have a document loaded when calling this api
     docViewer.on('documentLoaded', function() {
       instance.print();
+      // instance.print({ includeComments:true, includeAnnotations: true });
     });
   });
  */
 
 import { print } from 'helpers/print';
 import selectors from 'selectors';
 
-export default store => () => {
-  print(store.dispatch, selectors.isEmbedPrintSupported(store.getState()));
+export default store => options => {
+  const defaultOptions = {
+    allPages: false,
+    pagesToPrint: [],
+    includeAnnotations: false,
+    includeComments: false,
```

Comment:
Should `allPages` be `true` and `includeAnnotations` be `true` to match the default options in the print modal?